apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -574,6 +595,7 @@ export async function createBookingScenario(data: ScenarioData) {
     );
   }
   const eventTypes = await addEventTypes(data.eventTypes, data.users);
+  addHostsToDb(data.eventTypes);
```

Comment:
should we add await here ?

```suggestion
await  addHostsToDb(data.eventTypes);
```